src/corecel/sys/MemRegistry.hh

```diff
@@ -20,12 +20,12 @@ namespace celeritas
 //! SI prefix for multiples of 1024
 struct Kibi
 {
-    static CELER_CONSTEXPR_FUNCTION int value() { return 1024; }
+    static CELER_CONSTEXPR_FUNCTION long int value() { return 1024; }
```

Comment:
What the purpose of that change?  Rather than `long int`, do we mean `int64_t` ?  (And almost more important how to we remember to use the right type in similar future construct?)

Reply:
Int is 32-bit on windows and I wanted something "long"er. I try to steer away from special integer types for JSON I/O though now that I think about it, I'm not sure why. I'll unify all this.

Reply:
Because any memory change over 2GB overflows `int` on windows.

Reply:
it was the conversion between size_t and ptrdiff_t that was throwing me off. I've replaced it all with size_t.